gluten-arrow/pom.xml

```diff
@@ -47,7 +47,19 @@
   <dependencies>
     <dependency>
       <groupId>org.apache.gluten</groupId>
-      <artifactId>gluten-core</artifactId>
+      <artifactId>gluten-substrait</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Temporarily make `gluten-arrow` rely on `gluten-substrait`. In the future we'd merge `backends-velox` and `gluten-arrow` into a whole.

https://github.com/apache/incubator-gluten/issues/9639